models/engine/file_storage.py

```diff
@@ -68,3 +68,20 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """retrieve one object"""
+        if cls.__name__ in classes and id is not None:
+            objs = self.all(cls.__name__)
+            for key in objs.keys():
+                if key == cls.__name__ + '.' + id:
+                    return objs[key]
+        return None
+
+    def count(self, cls=None):
+        """count the number of objects in storage"""
+        if cls is None:
+            return len(self.__objects)
+        if cls.__name__ in classes:
+            return len(self.all(classes[cls.__name__]))
+        return 0
```

Comment:
You can do this code in max 3 lines using `len(obj)` function, where `obj` is our `clss`.

```suggestion
    def count(self, cls=None):
        """def that count the number of objects in storage"""
        return len(self.all(cls))
```

Reply:
Updated!